plugins/idea/src/main/java/com/oracle/plugin/jtreg/configuration/JTRegConfigurationRunnableState.java

```diff
@@ -107,6 +108,15 @@ protected JavaParameters createJavaParameters() throws ExecutionException {
         javaParameters.getProgramParametersList().clearAll();
         javaParameters.setMainClass("com.sun.javatest.regtest.Main");
 
+        if (JTRegUtils.IS_WINDOWS) {
```

Comment:
What happens if `System.getenv("TMP")` returns `null`?

Wouldn't it be better to check that first for each pair, and if the value is non-null, add the pair to the  `javaParameters` instance?